<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-DD4D3IC" name="TestEx" tests="4" failures="1" timestamp="17 Jul 2018 07:00:10 GMT" time="0.044" errors="0">
  <testcase name="test1" time="0.014" classname="com.reports.TestReport"/>
  <testcase name="test2" time="0.001" classname="com.reports.TestReport"/>
  <testcase name="test3" time="0.003" classname="com.reports.TestReport"/>
  <testcase name="testReport" time="0.002" classname="com.reports.TestReport">
    <failure type="java.lang.RuntimeException" message="this is my runtime exception">
      <![CDATA[java.lang.RuntimeException: this is my runtime exception
	at com.reports.TestReport.testReport(TestReport.java:30)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- testReport -->
</testsuite> <!-- TestEx -->
